apps/remote_control/test/lexical/remote_control/code_intelligence/rename_test.exs

```diff
@@ -0,0 +1,338 @@
+defmodule Lexical.RemoteControl.CodeIntelligence.RenameTest do
+  alias Lexical.Document
+  alias Lexical.Project
+  alias Lexical.RemoteControl
+  alias Lexical.RemoteControl.CodeIntelligence.Rename
+  alias Lexical.RemoteControl.Search
+
+  alias Lexical.Test.CodeSigil
+  alias Lexical.Test.CursorSupport
+  alias Lexical.Test.Fixtures
+
+  import CodeSigil
+  import CursorSupport
+  import Lexical.Test.EventualAssertions
+  import Fixtures
+
+  use ExUnit.Case
+
+  setup_all do
+    project = project()
+    RemoteControl.set_project(project)
+    start_supervised!({Document.Store, derive: [analysis: &Lexical.Ast.analyze/1]})
+
+    start_supervised!(
+      {Search.Store,
+       [
+         project,
+         fn _ -> {:ok, []} end,
+         fn _, _ -> {:ok, [], []} end,
+         Search.Store.Backends.Ets
+       ]}
+    )
+
+    assert_eventually Search.Store.loaded?()
+
+    {:ok, project: project}
+  end
+
+  setup %{project: project} do
+    uri = subject_uri(project)
+
+    on_exit(fn ->
+      Document.Store.close(uri)
+    end)
+
+    %{uri: uri}
+  end
+
+  describe "rename exact module" do
```

Comment:
nit: can you add the word `is` after the word `cusor` to all the test names?

Reply:
done.🤗🤗